readconfig.go

```diff
@@ -129,6 +130,14 @@ func (ReadConfig) Read() (QueueWorkerConfig, error) {
 		}
 	}
 
+	if val, exists := os.LookupEnv("disable_tls_certificate_validation"); exists {
```

Comment:
nie chcemy tutaj mieć wielkimi literami tego ENVa, jak to zazwyczaj jest w konwencji?

Reply:
spoko, to lepiej spójnie